index.html

```diff
@@ -224,6 +224,13 @@ <h3>Audio Content Hints</h3>
         this signal it may be appropriate to apply noise suppression or boost
         intelligibility of the incoming signal.
       </p></td></tr>
+      <tr><td><code id="idl-def-AudioContentHint.uniform-speech">uniform-speech</code></td><td><p>
+        The track should be treated as if it contains data for the purpose of
+        speech recognition by a machine. Consuming this signal it may be
+        appropriate to boost intelligibility of the incoming signal for
```

Comment:
What is the relevance of the term "intelligibility"? For machine or human? How is "intelligibility" measured by machine or human? Is that not for the speech processing algorithm, and ultimately the human that writes the speech recognition processing code to decide? Even then, the conclusion as to whether speech is "intelligibility" is arbitrary and subjective, depending on the context, the algorithm which processes articulation, tone, etc., the ability of the author of the code to understand the dialect in order to accurately. 

How exactly is the signal "intelligibility" determined? 

Is the goal is to actually apply a filtering algorithm for "transcription" client-side the term, suggest substituting 

"apply speech filtering and isolation algorithms to the incoming signal for transcription processing"

for 

"to boost intelligibility of the incoming signal for"

Reply:
On the other hand, "speech" input can be in the form of code https://www.w3.org/community/synthetic-media/wiki/Articulatory_Synthesis_Markup_Language, not necessarily audio output by a human, or audio output to speakers or a device, rather, a stream of markup, e.g., SSML; https://www.w3.org/community/synthetic-media/wiki/Articulatory_Synthesis_Markup_Language.